index.html

```diff
@@ -10,13 +11,14 @@
 </head>
 
 <body>
-  <div class="hero hero-bg is-fullheight">
+  <a href="#main-content" class="visually-hidden">Skip to Main content</a>
+  <header class="hero hero-bg is-fullheight">
     <div class="hero-head">
       <div class="navbar">
         <div class="container">
           <div class="navbar-brand">
-            <a class="navbar-item logo-link">
-              <img src="https://www.epam.com/content/dam/epam/global/logo_white-blue.svg">
+            <a href="#" class="navbar-item logo-link">
+              <img src="https://www.epam.com/content/dam/epam/global/logo_white-blue.svg" alt="epam home page">
             </a>
             <span class="navbar-burger burger" data-target="navbarMenu">
```

Comment:
Please fix the layout for the burger, because the current one doesn't work correctly.
```suggestion
<span class="navbar-burger burger" data-target="navbarMenu">
   <button class='burger-hidden' tabindex="0" aria-haspopup="true">Click here to open the menu</button>
     <nav id="navigation">
           <span></span>
           <span></span>
           <span></span>
     </nav>
 </span>
 
 === styles.scc ===
 .burger-hidden {
  border: 0;
  width: 50px;
  height: 50px;
  overflow: hidden;
  color: transparent;
  background-color: transparent;
}
```